packages/godwoken/src/client.ts

```diff
@@ -42,24 +43,36 @@ export class GodwokenClient {
     return +accountId;
   }
 
-  public async getScriptHashByShortScriptHash(
-    shortScriptHash: HexString
+  public async getRegistryAddressByScriptHash(
+    scriptHash: Hash,
+    registryId: U32
+  ): Promise<RegistryAddress | undefined> {
+    const registryAddress = await this.rpcCall(
+      "get_registry_address_by_script_hash",
+      scriptHash,
+      toHex(registryId)
+    );
+    return registryAddress;
+  }
+
+  public async getScriptHashByRegistryAddress(
+    registryAddress: HexString
```

Comment:
Is it possible to use [`RegistryAddress`](https://github.com/nervosnetwork/godwoken-web3/pull/282/files#diff-a1a6f34c884302fbf9b7dd717268a3fa8bbf49b1950af04c2e90cd9e028172c2R59) as the parameter type? "HexString" is a bit ambiguous.

Reply:
Renamed to `serializedRegistryAddress`